index.js

```diff
@@ -4,55 +4,152 @@ function addListeners() {
+        fadeOut(element, duration) {
+            element.style.transitionDuration = `${duration}ms`;
+            element.classList.remove('show');
+            element.classList.add('hide');
+        },
+
+        /**
+         * Функция, передвигающая элемент
+         * @param element — HTMLElement, который надо анимировать
+         * @param duration — Продолжительность анимации в миллисекундах
+         * @param translation — объект с полями x и y, обозначающими смещение блока
+         */
+        move(element, duration, translation) {
+            element.style.transitionDuration = `${duration}ms`;
+            element.style.transform = getTransform(translation, null);
+        },
+
+        /**
+         * Функция, увеличивающая/уменьшающая элемент
+         * @param element — HTMLElement, который надо анимировать
+         * @param duration — Продолжительность анимации в миллисекундах
+         * @param ratio — во сколько раз увеличить/уменьшить. Чтобы уменьшить, нужно передать значение меньше 1
+         */
+        scale(element, duration, ratio) {
+            element.style.transitionDuration =  `${duration}ms`;
+            element.style.transform = getTransform(null, ratio);
+        },
+        
+        /**
+         * Функция, передвигающая элемент на 100 пикселей вправо и на 20 вниз, а затем скрывающая его. Пропорция времени анимации 40/60
+         * @param element — HTMLElement, который надо анимировать
+         * @param duration — Продолжительность анимации в миллисекундах
+         */
+        moveAndHide(element, duration) {
+            const moveDuration = .4 * duration;
+            const hideDuration = .6 * duration;
+            this.move(element, moveDuration, {x: 100, y: 20});
+            
+            setTimeout(() => {
+                this.fadeOut(element, hideDuration);
+            }, moveDuration);
+        },
+        /**
+         * Функция, показывающая элемент и после паузы, скрывающая его. Каждая фаза длится 1/3 от указанной продолжительности анимации
+         * @param element — HTMLElement, который надо анимировать
+         * @param duration — Продолжительность анимации в миллисекундах
+         */
+        showAndHide(element, duration) {
+            const animationDuration = duration / 3;
+            const pauseDuration = 2 * animationDuration;
+            
+            this.fadeIn(element, animationDuration);
+            setTimeout(() => {
+                this.fadeOut(element, animationDuration);
+            }, pauseDuration);
+        },
+
+        heartBeating(element) {
+
+            const timer1 = setInterval(() => {
+                this.scale(element, 500, 1.4)
+            }, 500)
+
+            const timer2 = setInterval(() => {
+                this.scale(element, 500, 1)
+            }, 1000)
+
+            const stop = function () {
+                clearInterval(timer1);
+                clearInterval(timer2);
+            }
+
+            return {
+                stop: stop,
```

Comment:
Если название переменной совпадает с название поля объекта, в которое эта переменная должна быть сохранена, то можно писать так
```suggestion
                stop,
```